packaging/windows/install_script.nsis

```diff
@@ -156,12 +156,15 @@ FunctionEnd
 # Uninstaller
 Function un.onInit
     SetShellVarContext all
-
-    #Verify the uninstaller - last chance to back out
-    MessageBox MB_OKCANCEL "Permanantly remove ${APPNAME}? This will remove the WAL and the agent config." IDOK next
-        Abort
-    next:
-    !insertmacro VerifyUserIsAdmin
+    IfSilent ThisIsSilent WarnUser
+    ThisIsSilent:
+        Return
+    WarnUser:
+         #Verify the uninstaller - last chance to back out
+        MessageBox MB_OKCANCEL "Permanantly remove ${APPNAME}? This will remove the WAL and the agent config." IDOK next
+            Abort
+        next:
+        !insertmacro VerifyUserIsAdmin
```

Comment:
Should this macro still run even in silent mode?

Reply:
The macro VerifyUserIsAdmin will not run while running in silent mode, it will return early. If you are talking about un.OnInit function then there isn't a great way to stop it from running in silent mode.